### PR TITLE
Clarify empty icons aren't broken

### LINKED ISSUE #1191: Clarify that the empty boxes in icon view are filler for no icon, rather than being a broken icon
As reported in #1189 it's not clear from the UI if the empty boxes represent a broken image or not; we should make this clearer

### PR DESCRIPTION
**Pull request by Throne3d**
Fixes #1191 

Update, now looks like this:

image

Previously just shows an empty box:

image

Temporary intermediate one we discarded:

image

### CODE REVIEW

File: app/assets/stylesheets/characters.scss [hunk 1]
```diff
@@ -40,8 +40,13 @@
   border: 1px $border_color_character_iconless solid;
   width: 102px;
   height: 102px;
+  padding: 20px 5px;
   text-align: center;
 
+  color: $font_color_page;
```
Comment (Throne3d) [inline app/assets/stylesheets/characters.scss:46]: Without this, on styles that have a specific color for link hover, we see the text inside the icon change color on hover, which seems strange (it seems like it should be "fixed" as a property of the image).

**Comment by teceler** [comment]
Oh. don't like that. it looks incredibly Wrong xD

**Comment by Throne3d** [comment]
Any other suggestions? :D

**Comment by Throne3d** [comment]
Updated look:

image

**Pull request merged**